src/index.html

```diff
@@ -11,7 +11,220 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form
+      class="text-form"
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+    >
+      <fieldset class="fieldset">
+        <legend>Personal information</legend>
+        <div class="input">
+          <label>
+            Surname:
+            <input
+              type="text"
+              id="sname"
+              name="sname"
+              autocomplete="off"
+            >
+          </label>
+        </div>
+
+        <div class="input">
+          <label>
+            Name:
+            <input
+              type="text"
+              id="name"
+              name="name"
+              autocomplete="off"
+            >
+          </label>
+        </div>
+
+        <div class="input">
+          <label>
+            How old are You?
+            <input
+              type="number"
+              id="age"
+              name="age"
+              min="1"
+              max="100"
+              value="12"
+            >
+          </label>
+        </div>
+
+        <div class="input">
+          <label>
+            Full date of birth:
+            <input
+              type="date"
+              id="birthday"
+              name="birthday"
+            >
+          </label>
+        </div>
+
+        <div class="input">
+          <label>
+            I accept the term of the agreement
+            <input
+              type="checkbox"
+              id="terms"
+              name="terms"
+            >
+          </label>
+        </div>
+
+      </fieldset>
+
+      <fieldset class="fieldset">
+        <legend>Registration</legend>
+
+        <div class="input">
+          <label>
+            E-mail:
+            <input
+              type="email"
+              id="email"
+              name="email"
+              required
+              placeholder="email@example.com"
+            >
+          </label>
+        </div>
+
+        <div class="input">
+          <label>
+            Password:
+            <input
+              type="password"
+              id="passw"
+              name="passw"
+              minlength="6"
+              maxlength="32"
+              required
+            >
+          </label>
+        </div>
+
+      </fieldset>
+
+      <fieldset class="fieldset">
+        <legend>An interesting fact about you!</legend>
+        <div class="input">
+          <label>
+            Do you love cats?
+            <input
+              type="radio"
+              id="yes"
+              name="questionButton"
+              value="yes"
+            >
+          </label>
+          <label>
+            Yes
+            <input
+              type="radio"
+              id="no"
+              name="questionButton"
+              value="no"
+            >
+          </label>
+          <label>No</label>
```

Comment:
this labels don't work correctly
```suggestion
            Do you love cats?
            <label>
              <input
                type="radio"
                id="yes"
                name="questionButton"
                value="yes"
              >
              Yes
            </label>
            <label> 
              <input
                type="radio"
                id="no"
                name="questionButton"
                value="no"
              >
              No
            </label>
```

Reply:
If I choose 'yes' it chooses 'no'